examples/rcbilling-demo/playwright.config.ts

```diff
@@ -0,0 +1,97 @@
+import { fileURLToPath } from "url";
+import { dirname, resolve } from "path";
+import { defineConfig, devices } from "@playwright/test";
+import dotenv from "dotenv";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+dotenv.config({
+  path: resolve(__dirname, ".env"),
+});
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  globalSetup: (await import("path")).resolve("./playwright-global-setup.ts"),
+  testDir: "./src/tests",
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 4 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: process.env.CI
+    ? [["junit", { outputFile: "results.xml" }]]
+    : "list",
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    // baseURL: 'http://127.0.0.1:3000',
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: "on-first-retry",
+    ...(process.env.CI
+      ? {
+          headless: true, // Run in headless mode
+          viewport: { width: 1280, height: 720 }, // Set a smaller viewport
+          ignoreHTTPSErrors: true, // Ignore HTTPS errors
+          video: "off", // Disable video recording
+          screenshot: "off", // Disable screenshots
+          trace: "off", // Disable tracing
+        }
+      : {}),
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: "chromium",
+      use: { ...devices["Desktop Chrome"] },
+    },
+
+    {
+      name: "firefox",
+      use: { ...devices["Desktop Firefox"] },
+    },
+
+    // {
+    //   name: "webkit",
+    //   use: { ...devices["Desktop Safari"] },
+    // },
```

Comment:
webkit is disabled for now because CI is failing, but I'm working on changing the config to be able to re-enable this

Reply:
solved!